notebooks/nessie-iceberg-demo-nba.ipynb

```diff
@@ -0,0 +1,619 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Nessie Spark SQL Demo with NBA Dataset\n",
```

Comment:
No clue whether and how easy it is, but would it be possible to have a notebook that uses the `spark-sql` REPL? Not asking for that for this PR though.

Reply:
its possible, but a bit more of a faf to set up. Definitely for a follow up